pkg/cmd/roachtest/tests/mixed_version_sql_stats.go

```diff
@@ -84,7 +84,7 @@ func runSQLStatsMixedVersion(ctx context.Context, t test.Test, c cluster.Cluster
 
 	for name, reqType := range requestTypes {
 		mvt.InMixedVersion("request "+name, func(ctx context.Context, l *logger.Logger, rng *rand.Rand, h *mixedversion.Helper) error {
-			return runGetRequests(ctx, c, l, c.CRDBNodes(), reqType)
+			return runGetRequests(ctx, c, l, h.AvailableNodes(), reqType)
```

Comment:
nit: do we need this anymore since we disabled panics around hook steps? Should be harmless in theory though.